src/cli.rs

```diff
@@ -0,0 +1,11 @@
+use clap::Parser;
+
+#[derive(Parser, Debug)]
+#[command(version, about, long_about = None)]
+pub struct Args {
+    #[arg(short, long, action = clap::ArgAction::SetTrue)]
+    pub client: Option<bool>,
+
+    #[arg(long, default_value = "master.ron")]
+    pub config_file: String,
+}
```

Comment:
this file has been removed there https://github.com/Hyle-org/hyle/pull/76 
I think it's not used